README.md

```diff
@@ -2,63 +2,34 @@
 
-### Env variables
+### Slack alerts
 
-```sh
-export SEAL_ORGANISATION="your_github_organisation"
-export GITHUB_TOKEN="get_your_github_token_from_yourgithub_settings"
-export GITHUB_API_ENDPOINT="your_github_api_endpoint" # OPTIONAL If you are using a Github Enterprise instance
-export SLACK_WEBHOOK="get_your_incoming_webhook_link_for_your_slack_group_channel"
-export SLACK_CHANNEL="#whatever-channel-you-prefer"
-export GITHUB_USE_LABELS=true
-export GITHUB_SECURITY_ALERTS=true
-export GITHUB_EXCLUDE_LABELS="[DO NOT MERGE],Don't merge,DO NOT MERGE,Waiting on factcheck,wip"
-export GITHUB_REPOS="myapp,anotherrepo" # Repos you want to be notified about
-export COMPACT=true # Use a more compact version of the seal output
-export SEAL_QUOTES="Everyone should have the opportunity to learn. Don't be afraid to pick up stories on things you don't understand and ask for help with them. Try to pair when possible."
-```
+To customize which alerts your team channel gets, find your team in [config/alphagov.yml](https://github.com/alphagov/seal/blob/main/config/alphagov.yml) and set the following values to `true` or `false`:
 
-- To get a new `GITHUB_TOKEN`, head to: https://github.com/settings/tokens
-- To get a new `SLACK_WEBHOOK`, head to: https://slack.com/services/new/incoming-webhook
-
-### GitHub Actions
-
-This repository now utilizes GitHub Actions for automated Slack notifications. The workflows are defined in the `.github/workflows` directory and perform various tasks at different times of the day:
-
-- [Morning Seal](https://github.com/alphagov/seal/blob/main/.github/workflows/morning_seal.yml): Runs the Seal bot in the morning, posting about old and recent pull requests by team members and also quotes.
-- [Afternoon Seal](https://github.com/alphagov/seal/blob/main/.github/workflows/afternoon_seal.yml): Runs the Seal bot in the afternoon, posting quotes.
-- [Dependapanda](https://github.com/alphagov/seal/blob/main/.github/workflows/dependapanda.yml): Runs the Dependapanda bot in the morning, posting about Dependabot pull requests and security information.
-
-The Morning Seal and Dependapanda workflows run every weekday morning, while the Afternoon Seal workflow runs every weekday afternoon.
-
-To customize which bots post to your team channel, add or remove your team name in the workflows above. The team name should correspond to a key in [config/alphagov.yml](https://github.com/alphagov/seal/blob/main/config/alphagov.yml), which should have a `channel` property denoting the name of your team's Slack channel.
-
-The `channel` property should match a channel name in the [list of repos by ownership in the Developer Docs](https://docs.publishing.service.gov.uk/repos.html#repos-by-team), otherwise Dependapanda won't know which repos to notify your channel about.
+- morning_seal_quotes: Morning quotes set by your team
+- morning_seal_prs: Morning alerts about old and recent pull requests by team members
```

Comment:
Another quick refactor suggestion: I'd rename this `seal_prs` (as there is no 'afternoon' equivalent, and for consistency - it's `dependapanda` rather than `morning_dependapanda`) and move this below `afternoon_seal_quotes` in this list.